gitpod-io/gitpod
# [ws-manager-mk2] Enable by default in preview envs

## Description

Enables mk2 by default in preview envs

## Related Issue(s)

Fixes #

## How to test


Create preview env without `GITPOD_WSMANAGER_MK2=true`, and confirm mk2 gets deployed.

Creating a preview env with `GITPOD_WSMANAGER_MK2=false` should deploy mk1

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - wv-mk2-enaf7c31f7e8b</li>
	<li><b>🔗 URL</b> - <a href="https://wv-mk2-enaf7c31f7e8b.preview.gitpod-dev.com/workspaces" target="_blank">wv-mk2-enaf7c31f7e8b.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold
